tests/common.py

```diff
@@ -309,16 +309,22 @@ def to_tmpf(tmpd):
     return os.path.join(tmpd, "nqv248p90")
 
 
-def are_splines_equal(a, b):
+def are_splines_equal(a, b, print_=True):
```

Comment:
Should we not try to reduce the printed output? This function is only for testing but still, I would normally expect options as these to be `false` by default.
```suggestion
def are_splines_equal(a, b, print_=False):
```